### DISCUSSION TITLE
How can i scale my temporal, so that it can handle more workers.

### DISCUSSION THREAD

**Discussion by manojkumargogula** [alexandrevilain/temporal-operator#709] [Q&A] [answered]
@alexandrevilain  i can able to run less than 5 workers at a time with using the single temporal pod, how do it scale is up.

**Comment by alexandrevilain** [comment, marked as answer]
Hi @manojkumargogula !

What are you trying to scale ? Your worker or the temporal services ? 
For your worker, this is your own system, like a `Deployment`.
And for temporal services, your can check services replicas on the `TemporalCluster` resource. You can check https://temporal-operator.pages.dev/api/v1beta1/#temporal.io/v1beta1.ServicesSpec

**Reply by manojkumargogula** [reply]
Thank @alexandrevilain, you made my day.
